src/login.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sabka Bazaar</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Dosis:400,600,700">
+</head>
+
+<body>
+    <header id="header">
+    </header>
+    <section>
+        <div class="row">
+            <div class="login">
+                <div class="login__text">
+                    <h2>Login</h2>
+                    <p>Get acces to your Order, Wishlists and Recommendations</p>
+                </div>
+                <div class="login__form">
+                    <form autocomplete="off" method="POST">
+                        <div class="email">
+                            <input type="email" required>
+                            <span class="highlight"></span>
```

Comment:
Don't create DOM for styling use pseudo elements before and after instead.

Reply:
Removed. Done